tests/api_gov.spec.js

```diff
@@ -360,4 +362,45 @@ describe('TESTING /v2/gov/new zealand', () => {
         });
     });
   });
-});
+
+describe('TESTING /v2/gov/south africa', () => {
+    it('/v2/gov/south africa correct data', (done) => {
+      chai.request(app)
+        .get('/v2/gov/south africa')
+        .end((err, res) => {
+          testBasicProperties(err, res, 200, 'object');
+          res.body.should.have.property('updated');
+          res.body.should.have.property('national');
+		  res.body.should.have.property('provinces');
+
+		  res.body.national.timeline.length.should.be.at.least(107);
+
+		  let dayNational = res.body.national.timeline.find(d => d.date === '2020-06-02');
```

Comment:
use `const` since this variable won't change

Reply:
We can do those too though